src/app/hooks/useLogin.ts

```
@@ -0,0 +1,126 @@
+      } catch (err) {
+        setError(err instanceof Error ? err : new Error("An unknown error occurred"));
+        throw err;
+      } finally {
+        setIsLoading(false);
+      }
+    };
+  
+    const mutate = (
+      data: { token: string; newPassword: string },
+      options?: {
+        onSuccess?: (data: any) => void;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)